pages/downloads.md

```diff
@@ -85,26 +85,29 @@ available in a future update to this site.
 {% for srcbin in srcbinArray %}
 {% assign lnkFile = 'accumulo-' | append: linkVers | append: '-' | append: srcbin | append: '.tar.gz' %}
 {% assign lnkSuffix = '/accumulo/' | append: linkVers | append: '/' | append: lnkFile %}
-<div class="row btn-group" style="margin-left: 20px; margin-bottom: 5px; display: block">
-  <div class="col btn-group">
+<div class="row" style="margin-left: 20px; margin-bottom: 5px;">
+  <div class="btn-group col-auto">
     <a {{btnDownloadStyle}} href="{{closerLink}}{{lnkSuffix}}" link-suffix="{{lnkSuffix}}">{{lnkFile}}{{glyphSave}}</a>
-  </div><div class="col btn-group">
-    <a {{btnSigStyle}} href="{{downloadsLink}}{{lnkSuffix}}.asc">ASC{{glyphLock}}</a>
-    <a {{btnHashStyle}} href="{{downloadsLink}}{{lnkSuffix}}.sha512">SHA{{glyphLock}}</a>
+    <div class="btn-group">
+      <a {{btnSigStyle}} href="{{downloadsLink}}{{lnkSuffix}}.asc">ASC{{glyphLock}}</a>
+      <a {{btnHashStyle}} href="{{downloadsLink}}{{lnkSuffix}}.sha512">SHA{{glyphLock}}</a>
+    </div>
   </div>
 </div>
 {% endfor %}
-<div class="row btn-group-sm" style="margin: 20px">
-  <a {{btnDocStyle}} href="{{site.baseurl}}/release/accumulo-{{linkVers}}">Release Notes</a>
-  <a {{btnDocStyle}} href="https://github.com/apache/accumulo/blob/rel/{{linkVers}}/README.md">README</a>
-  <a {{btnDocStyle}} href="{{site.baseurl}}/docs/2.x">Online Documentation</a>
-  <a {{btnDocStyle}} href="https://github.com/apache/accumulo-examples">Examples</a>
-  <a {{btnDocStyle}} href="{{site.baseurl}}/docs/2.x/apidocs3">Java API</a>
+<div class="row" style="margin: 20px;">
+  <div class="btn-group-sm">
```

Comment:
This extra div creates a single column inside the row div. Previously, the intent was to have each `<a>` element be a separate column within the row.

Reply:
In a713543 I removed the unneeded `row` div that wrapped the `btn-group-sm` div and it seems to work well.

Reply:
Yeah, that change should be fine. The div with the `btn-group-sm` is already acting as the "row" container anyway.